configs/application/config.json

```
@@ -6,38 +6,39 @@
 			"initialStores": [
 				{
 					"name": "Finsemble-AppLauncher-Store",
+					"preferPreviousState": true,
```

Do we need this to be true here?
```suggestion
					"preferPreviousState": false,
```